lib/azure/armrest/armrest_service.rb

```diff
@@ -362,49 +374,64 @@ def self.raise_api_exception(e)
 
       def rest_get(url)
         self.class.rest_get(
-          url,
-          :accept        => armrest_configuration.accept,
-          :content_type  => armrest_configuration.content_type,
-          :authorization => armrest_configuration.token,
+          :url     => url,
```

Comment:
for the instance methods (`rest_get`, `rest_put` etc) they all look very similar. We should be able to extract the composing hash logic into a private helper method